tests/expectations/metrics/test_core.py

```diff
@@ -125,7 +196,10 @@ def test_stdev_metric_pd():
         metrics_to_resolve=(desired_metric,), metrics=metrics
     )
     metrics.update(results)
-    assert results == {desired_metric.id: 1}
+    assert results == {desired_metric.id: result}
```

Comment:
@TrangPham The previous assertion was better (in my opinion), because it states explicitly what we expect; if we say `result`, the `result` may be wrong, but the assertion will give `True`.  Thanks.

Reply:
I think we are good here - the `result` comes from the parametrization so this assertion is checking against the value provided for this test case (`1` in this case).

Reply:
@anthonyburdi How are you seeing the `result` coming from parametrization?  I am seeing it come from resolving a metric (in line 195); or is it guaranteed thanks to line 167?  Not sure without deeper thinking that it is "necessarily" guaranteed (things can go wrong while computing the metric).  In our style of tests in this module, I believe that we compare explicitly -- "just in case" -- happy to discuss what is best.  Thank you.

Reply:
I have updated the variable name to be less confusing and more readable.